### PR TITLE
Added Windows example in README.rst

### PR DESCRIPTION
**Pull request by sackh**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -3,9 +3,10 @@ mcinstall
 
 A script to quickly make/provision a fresh Miniconda installation from scratch.
 
-This will download a Miniconda binary from https://repo.continuum.io/miniconda/
-like https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh,
-unpack and install it locally. It was tested on macOS and Linux ok, and has
+This will download a Miniconda binary based on your operating system from https://repo.continuum.io/miniconda/
```
Comment (deeplook) [inline README.rst]: I'm planning to add support for the existing (although no longer maintained) BerryConda distribution for ARM 6 and 7 which would not be downloaded from continuum.io. So I would not stress this so much here.

File: README.rst [hunk 2]
```diff
@@ -42,6 +43,20 @@ This can be used then with a command like this::
 
     source ~/Downloads/mc3/bin/activate
 
+Windows Example:
+
+On Windows if you run like this::
+
+    python3 mcinstall.py %USERPROFILE%\downloads\mc3
```
Comment (deeplook) [inline README.rst]: This refers to using the script inside the repo without pip-installing it. Can we either make this clear or align with the other examples which use the pip-installed tool named `mcinstall` without extension?

Comment (sackh) [reply README.rst]: sure will change this

File: README.rst (hunk 1 above)
Comment (sackh) [reply README.rst]: ok. I will remove this and will only keep OS specific links for now.

**Pull request merged**